client/src/app/pages/applications/manage-imports-details/manage-imports-details.tsx

```diff
@@ -74,6 +74,7 @@ export const ManageImportsDetails: React.FC = () => {
   const tableControls = useLocalTableControls({
     tableName: "manage-imports-details",
     idProperty: "Application Name",
```

Comment:
`idProperty: "id"`

Reply:
(resolving this comment and responding the other comment -- it will be the same discussion)